doc/source/groups_of_atoms.rst

```diff
@@ -0,0 +1,117 @@
+.. -*- coding: utf-8 -*-
+.. _groups-of-atoms:
+
+Groups of atoms
+===============
+
+MDAnalysis has a hierarchy of :class:`~MDAnalysis.core.groups.Atom` containers that are used throughout the code.
+
+.. image:: images/classes.png
+
+First and foremost is the :class:`~MDAnalysis.core.groups.AtomGroup`. An :class:`~MDAnalysis.core.groups.AtomGroup` is the primary :class:`~MDAnalysis.core.groups.Atom` container; virtually everything can be accessed through it, as detailed `above <https://www.mdanalysis.org/UserGuide/data_structures.html#atomgroup>`_. This includes chemically meaningful groups of :class:`~MDAnalysis.core.groups.Atom`\ s such as a :code:`Residue` or a :code:`Segment`. 
```

Comment:
Don't use the inline link for "above", use a sphinx ref, which will never break.